lego/apps/users/views/memberships.py

```diff
@@ -9,6 +11,12 @@
 class MembershipViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
     serializer_class = MembershipSerializer
     filter_class = MembershipFilterSet
+    ordering_fields = ["id", "role", "user__first_name"]
```

Comment:
Can we rename `user__first_name` to something a bit shorter/human?

Reply:
Not gonna use it, so ill just remove it